lib/src/modules/escapeButton/EscapeButton.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ButtonIcon } from 'lib';
+
+export function EscapeButton({ onClick }) {
+  return (
+    <span className="escape-button">
+      <ButtonIcon onClick={onClick} name="cross" />
+    </span>
+  );
+}
```

Comment:
💚 Could we add a cheeky bonus extra to this, and add an optional esc key handler 👀  the name might just be throwing me off with this, if thats not valid maybe 'CloseButton' might be better?